tests/Queue/QueueSizeTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Queue;
+
+use Queue;
```

Comment:
You need to reference the actual facade class:
```suggestion
use Illuminate\Support\Facades\Queue;
```